src/SonarScanner.MSBuild.Tasks/GetAnalyzerSettings.cs

```diff
@@ -80,7 +80,6 @@ public class GetAnalyzerSettings : Task
         /// <summary>
         /// The language for which we are gettings the settings
         /// </summary>
-        [Required]
```

Comment:
Root cause: _Required_ was added in 4.4.1. However, the targets will be called for non-C#/VB projects and so need to handle _Language_ not being set gracefully.